hackathon_site/hackathon_site/urls.py

```diff
@@ -45,14 +45,21 @@
 
 if settings.DEBUG:
     import debug_toolbar
+    from django.core.exceptions import ImproperlyConfigured
     from django.urls import re_path
     from django.views.static import serve
 
     urlpatterns = [path("__debug__/", include(debug_toolbar.urls))] + urlpatterns
 
+    if settings.MEDIA_URL.startswith("http"):
```

Comment:
I'm confused why it matters if the `MEDIA_URL` starts with http

Reply:
You can set `MEDIA_URL` to something off-site if you serve your user-uploaded files on a different domain (eg because they go to an s3 bucket). In that case you won't be able to serve them in debug mode which is what the snippet here does, so I raise an `ImproperlyConfigured` to warn the user about that so they remove that part of the URL conf